### PR TITLE
Oo sunu #20 updating to pain 008 001 007

### PR DESCRIPTION
**Pull request by mcmpp**

### CODE REVIEW

File: src/main/scala/ch/openolitor/buchhaltung/BuchhaltungRoutes.scala [hunk 1]
```diff
@@ -124,8 +123,9 @@ trait BuchhaltungRoutes extends HttpService with ActorReferences
...
               entity(as[RechnungenContainer]) { cont =>
                 onSuccess(buchhaltungReadRepository.getByIds(rechnungMapping, cont.ids)) { rechnungen =>
-                  val xml = generatePain008(rechnungen)
-                  val stream = new ByteArrayInputStream(xml.getBytes(StandardCharsets.UTF_8))
+                  val xml = generatePain008_001_07(rechnungen)
+                  logger.debug(s"=================================================================== xml = $xml")
+                  //val stream = new ByteArrayInputStream(xml.getBytes(StandardCharsets.UTF_8))
                   //storeToFileStore(RechnungExportDaten, Some("exportFile"), stream, "exportFile.xml") { (id, metadata) =>
                   //  complete("File uploaded")
                   //}
```
Comment (AlwinEgger) [inline src/main/scala/ch/openolitor/buchhaltung/BuchhaltungRoutes.scala]: Is this part of another task?

File: src/main/scala/ch/openolitor/buchhaltung/rechnungsexport/iso20022/Pain008_003_02_Export.scala [hunk 2]
```diff
@@ -69,17 +70,17 @@ class Pain008_003_02_Export extends LazyLogging {
...
+  //jval CtrlSum = None
+  //jval partyIdentificationSepa1 = kontoDatenProjekt.creditorIdentifier.getOrElse("Initiator")
 
-    GroupHeaderSDD(MsgId, CreDtTm, NbOfTxs, CtrlSum, PartyIdentificationSEPA1(Some(partyIdentificationSepa1), None))
-  }
+  //jGroupHeaderSDD(MsgId, CreDtTm, NbOfTxs, CtrlSum, PartyIdentificationSEPA1(Some(partyIdentificationSepa1), None))
+  //}
 
-  private def getPaymentInstructionInformationSDD(rechnung: Rechnung, kontoDatenKunde: KontoDaten, kontoDatenProjekt: KontoDaten, NbOfTxs: String): PaymentInstructionInformationSDD = {
+  /*private def getPaymentInstructionInformationSDD(rechnung: Rechnung, kontoDatenKunde: KontoDaten, kontoDatenProjekt: KontoDaten, NbOfTxs: String): PaymentInstructionInformationSDD = {
```
Comment (AlwinEgger) [inline src/main/scala/ch/openolitor/buchhaltung/rechnungsexport/iso20022/Pain008_003_02_Export.scala]: Cleanup if not needed

Comment (mcmpp) [reply src/main/scala/ch/openolitor/buchhaltung/rechnungsexport/iso20022/Pain008_003_02_Export.scala]: done, the whole file needs to go. This is the old version

File: src/main/scala/ch/openolitor/buchhaltung/BuchhaltungRoutes.scala (hunk 1 above)
Comment (mcmpp) [reply src/main/scala/ch/openolitor/buchhaltung/BuchhaltungRoutes.scala]: done. Next story add the functionality here

**Pull request merged**